lib/asciidoctor.rb

```diff
@@ -7,10 +7,9 @@
 
 require 'set'
 
-# NOTE RUBY_ENGINE == 'opal' conditional blocks are filtered by the Opal preprocessor
+# NOTE RUBY_ENGINE == 'opal' conditional blocks like this are filtered by the Opal preprocessor
 if RUBY_ENGINE == 'opal'
-  # NOTE asciidoctor/opal_ext is supplied by the Asciidoctor.js build
-  require 'asciidoctor/opal_ext'
+  require 'asciidoctor/opal' # supplies compile and runtime overrides for the Opal environment; fulfilled by Asciidoctor.js
```

Comment:
Basically the goal of the first `require` is to provide/override Ruby method (provided by Opal) and the goal of the second `require` is to provide/override Asciidoctor method.

Maybe we could create a module named `js` ?

```ruby
require 'asciidoctor/js/opal'
```

```ruby
require 'asciidoctor/js/asciidoctor'
```
(I don't like the repetition of `asciidoctor`)

> I'm open to other names as well. Ideas I had include coda, postscript, post, overrides, patch, annex, omega. I kind of like "postscript" and "omega".

I like `omega` but at the same time I think it's the least understandable